### PR TITLE
swagger documentation

### PR DESCRIPTION
**Pull request by rajatxooa**

### CODE REVIEW

File: balance-transfer-java/README.md [hunk 1]
```diff
@@ -62,4 +62,21 @@ This method expects an argument `[ "<account>" ]`.
 
 6. Navigate to **Transactions** tab.
 
-7. You can expand the data field to see your transactions.
\ No newline at end of file
+7. You can expand the data field to see your transactions.
+
+
+## What is Swagger and how it is used by Xooa.
```
Comment (sandeepxooa) [inline balance-transfer-java/README.md]: Change HEADING what is swagger.json and purpose

File: balance-transfer-java/README.md [hunk 2]
```diff
@@ -62,4 +62,21 @@ This method expects an argument `[ "<account>" ]`.
...
+
+Swagger is used to simplify API development and is used by Xooa for defining REST APIs. 
+Using swagger Xooa provides you an interface to make API requests.
+Place **swagger.json** file along with your Smart Contract and Xooa will pick it up while deploying and will integrate it with your app.
+
+### Steps to access swagger
+
```
Comment (sandeepxooa) [inline balance-transfer-java/README.md]: Navigate to API documentation

File: balance-transfer-java/README.md (hunk 1 above)
Comment (rajatxooa) [reply balance-transfer-java/README.md]: Ok

File: balance-transfer-java/README.md (hunk 2 above)
Comment (rajatxooa) [reply balance-transfer-java/README.md]: ok

**Pull request merged**